src/open_inwoner/openzaak/tests/test_documents.py

```diff
@@ -179,6 +179,17 @@ def test_document_content_is_retrieved_when_user_logged_in_via_digid(self, m):
             response.headers["Content-Length"], str(len(self.informatie_object_content))
         )
 
+    def test_document_retrieval_logs_case_identification_and_file(self, m):
```

Comment:
Earlier I appended the logging test to the general test, but it is more readable to have them separate and not test multiple things at same time.